app/views/api/feedback_submissions/create.json.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.feedback_submission do
+  json.body @feedback.body
```

Comment:
Wasn't sure what to render for the success case, in theory a 201 response code should be all is needed, but returning some content seemed to me more consistent. Left the id out since there is no show action